charts/trusted-artifact-signer/values.yaml

```diff
@@ -5,8 +5,15 @@ global:
   appsSubdomain: ""
 
 configs:
+  pull_secret:
+    secret_create: true
+    value: {"auths":{"cloud.openshift.com":{"auth":"b3BlbnNoaWZ0LXJlbGVhc2UtZGV2K29jbV9hY2Nlc3NfMTNiZmY3NjNiNmFiNDkwYjgwYWJiZTI0OWE4OWJjNGE6TExPVVlPTE9BTk1EMDJWR1BXSVZKUUxYVzVJVFZVR0tIVjhHOVUyUzlUNzJEVE1VTTMzRkZQNUswUTZOWkdSRw==","email":"grpereir@redhat.com"},"quay.io":{"auth":"b3BlbnNoaWZ0LXJlbGVhc2UtZGV2K29jbV9hY2Nlc3NfMTNiZmY3NjNiNmFiNDkwYjgwYWJiZTI0OWE4OWJjNGE6TExPVVlPTE9BTk1EMDJWR1BXSVZKUUxYVzVJVFZVR0tIVjhHOVUyUzlUNzJEVE1VTTMzRkZQNUswUTZOWkdSRw==","email":"grpereir@redhat.com"},"registry.connect.redhat.com":{"auth":"fHVoYy1wb29sLTU1ODc1NmZkLTNjZTctNGUzMC1hMTE5LTc1NmEyZjEwZDRkZTpleUpoYkdjaU9pSlNVelV4TWlKOS5leUp6ZFdJaU9pSXdOREpqTXpBMVl6azVOVEkwWXpOaU9HRXhPV0ZoT1RBMk9EaGtNbVprT0NKOS5maXBCYXcyOXVpdU5WTk9sVndfa0lhWlRndkdJU3J3TWhJVUpackNCblM1UTBSRmlPbFRzTlVab0Nob1g3SFVyekVkX1ZUWHRnT1FoZ1QxR1FkWFNrOWQ5dzk5akZDd3pvOE5FVzYyMWZmZTh1RHVLR3c4NFpjTm1kMlo3ZDl3OHJGZXBFa3dUeTBuVzlnMnoyWEtnOGJhYmZaSUJoR244U24tM2g1MUxWdEhBbTVSbEN6T1ZpdGdhdk9YdUd2ZHdibU5GdUFZODJ3d3NqZWVkZDVFQVlFTm5UZHk3ZlNaWmdIY0FYN1l3UkNNRXVnNjNVVzR4TGZPdEhtTkJrWDY5SXc3bFY5WklUcjNBY3ZUSGswZ0JWSmswVGJHSHFWeko4S2hYTWRSVDRmMHNYZUpGWm5sMHMzbUJTajNtTUxSSHF6UHZzeVBxT3o2YWpubXUxTnJBeEFhb1F1VXloT1pyMjd6cVQ4SjlnTzNRVllDTXdHQ0xpeGlhRzhEd1R2cTF3eGl4dnRzcWVENkhVck1EemtSYnVQYnh0V3hPTmpZNnJDWVlCV3BxQUdlNlg0SnZEbXB0U3Y2ZjZ5THBieDRfLVBDN0Z0TzB1Nk9MSFlRTUpaSUhEYXRkOXJuakNhb3JkVzJxTldZTm5mczZKbHotMy1taGFWSG1BSjBYcmpVdGQ5UFBFREhZQndrb29XZUdIYjExU2xfY2RhdklPdU02NHBickkwR1dRXzJ2LWxCWmd4UzVzNFRoWURrMkUyOHRTd1ZZR3A0ckx5dXdGVHphSnQ5TVcxcms2ZW5ndGhQbEt6cnphTTBMWG51TWVyUmgyRXlaNkVyN1drUnZTN2p4U1lRVDQxM0lUdmNtcUZoM0UtSFdBXzB4SzZ2eUNyaVpLMGVxRWotT1RpSQ==","email":"grpereir@redhat.com"},"registry.redhat.io":{"auth":"fHVoYy1wb29sLTU1ODc1NmZkLTNjZTctNGUzMC1hMTE5LTc1NmEyZjEwZDRkZTpleUpoYkdjaU9pSlNVelV4TWlKOS5leUp6ZFdJaU9pSXdOREpqTXpBMVl6azVOVEkwWXpOaU9HRXhPV0ZoT1RBMk9EaGtNbVprT0NKOS5maXBCYXcyOXVpdU5WTk9sVndfa0lhWlRndkdJU3J3TWhJVUpackNCblM1UTBSRmlPbFRzTlVab0Nob1g3SFVyekVkX1ZUWHRnT1FoZ1QxR1FkWFNrOWQ5dzk5akZDd3pvOE5FVzYyMWZmZTh1RHVLR3c4NFpjTm1kMlo3ZDl3OHJGZXBFa3dUeTBuVzlnMnoyWEtnOGJhYmZaSUJoR244U24tM2g1MUxWdEhBbTVSbEN6T1ZpdGdhdk9YdUd2ZHdibU5GdUFZODJ3d3NqZWVkZDVFQVlFTm5UZHk3ZlNaWmdIY0FYN1l3UkNNRXVnNjNVVzR4TGZPdEhtTkJrWDY5SXc3bFY5WklUcjNBY3ZUSGswZ0JWSmswVGJHSHFWeko4S2hYTWRSVDRmMHNYZUpGWm5sMHMzbUJTajNtTUxSSHF6UHZzeVBxT3o2YWpubXUxTnJBeEFhb1F1VXloT1pyMjd6cVQ4SjlnTzNRVllDTXdHQ0xpeGlhRzhEd1R2cTF3eGl4dnRzcWVENkhVck1EemtSYnVQYnh0V3hPTmpZNnJDWVlCV3BxQUdlNlg0SnZEbXB0U3Y2ZjZ5THBieDRfLVBDN0Z0TzB1Nk9MSFlRTUpaSUhEYXRkOXJuakNhb3JkVzJxTldZTm5mczZKbHotMy1taGFWSG1BSjBYcmpVdGQ5UFBFREhZQndrb29XZUdIYjExU2xfY2RhdklPdU02NHBickkwR1dRXzJ2LWxCWmd4UzVzNFRoWURrMkUyOHRTd1ZZR3A0ckx5dXdGVHphSnQ5TVcxcms2ZW5ndGhQbEt6cnphTTBMWG51TWVyUmgyRXlaNkVyN1drUnZTN2p4U1lRVDQxM0lUdmNtcUZoM0UtSFdBXzB4SzZ2eUNyaVpLMGVxRWotT1RpSQ==","email":"grpereir@redhat.com"}},"orgId":"11009103","userId":"54342237"}
```

Comment:
I think that this should be empty. You probably by mistake commit your pull secret credentials.